pyERGM/metrics.py

```diff
@@ -175,6 +174,16 @@ def _get_ignored_features(self):
 
         return ignored_features
 
+    def _calc_bootstrapped_scalar_feature(self, first_halves_to_use: np.ndarray,
+                                          first_halves_indices: np.ndarray[int], second_halves_indices: np.ndarray[int],
+                                          max_feature_val_calculator: callable):
```

Comment:
How come this doesn't fail? What is `callable`?
I don't see an import from `typing` or an alias for `typing.Callable`